schemas/index.js

```diff
@@ -0,0 +1,13 @@
+import * as walletAddresses from "./walletAddresses.js";
+
+export const schemas = new Map([
+  [
+    "walletAddresses",
+    {
+      model: walletAddresses,
+      modelType: "AvroBinary",
+      payloadLocation: "Itemized",
+      settings: ["SignatureRequired"],
+    },
```

Comment:
Do we need `AppendOnly` for itemized schema ?

Reply:
In this case it is OK for items to be removed without issue.